src/electron/device-connect-view/Styles/deviceconnectview.scss

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../../common/styles/common.scss';
+
+body {
+    -webkit-app-region: drag;
+    margin: 0px;
+}
+
+button {
+    -webkit-app-region: no-drag;
+}
+
+footer {
```

Comment:
Consider using per-component styles for the styles that are specific to a given react component, rather than one big style document for the whole page; that's the model we're trying to move to for newer components. `/src/reports/components/` has a lot of examples of this to take a look at

Reply:
Updated to use per-component styles